### ISSUE TITLE
Powershell reference is not complete

### ISSUE THREAD

**Issue by alpaix** [NuGet/Home#2107]
[Package Manager Console Powershell Reference](https://docs.nuget.org/consume/package-manager-console-powershell-reference) should include among others:
- Command and argument aliases where applicable;
- Position;
- Default value;
- Whether cmdlet accepts pipeline input.

For reference, see typical documentation page here https://technet.microsoft.com/en-us/library/hh849901.aspx.

**Comment by yishaigalatzer** [comment]
@alpaix please provide more details, what reference are you referring to (docs.nuget.org ? somewhere else)

**Comment by alpaix** [comment]
@yishaigalatzer Updated description with official NuGet documentation page link.

Also attaching an image from [TechNet](https://technet.microsoft.com/en-us/library/hh849901.aspx) as a reference example for full parameters documentation:
![image](https://cloud.githubusercontent.com/assets/8239356/13334250/f498beae-dbbf-11e5-9b06-6bfa0bcb77a3.png)

**Comment by kraigb** [comment]
@alpaix I've completed an overhaul of the reference, see https://docs.microsoft.com/en-us/nuget/tools/powershell-reference, which has the different commands broken out into individual topics.

I didn't go quite so far as to match the references you point. First, I see the open sentences on those references pages and the description sections redundant, so the description is simply at the top. Second, there is very little variance with parameters among the NuGet commands--only a few are required, only a few have default values, they are almost always named instead of positional, and none of them accept pipeline input or wildcards. For this reason, I list the arguments (with the occasional "required" and default value) in more concise tables; investing the time to break them out into the extended details would add a lot of text and visual noise without, in my opinion, adding any value. 

Those details are, by the way, available by doing _get-help <command> -full_, which is what I used to reconcile the arguments and their details.

I hope this is satisfactory. If not, it's worth noting that breaking out all the extended details to exactly match the TechNet references will be a significantly lower priority for the docs compared with many other extant issues.
